static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-nav.js

```diff
@@ -0,0 +1,158 @@
+define([
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/mediator'
+], function (
+    $,
+    _,
+    config,
+    detect,
+    mediator
+) {
+    return function () {
+        this.id = 'MtStickyNav';
+        this.start = '2015-03-19';
+        this.expiry = '2015-05-19';
+        this.author = 'Zofia Korcz';
+        this.description = 'Top navigation and top ad slot are sticky';
+        this.audience = 0.01;
+        this.audienceOffset = 0.3;
```

Comment:
From what I noticed in https://frontend.gutools.co.uk/analytics/abtests only 'AcrossTheCountry' may overlap my test but it's related to the containers not navigation/top slot so I don't think that it may cause an issue. 
What I need to worry the most is that MtStickyNav won't overlap any other tests related to the top slot, so MtTopBelowNav and MtTopBelowFirstContainer but that's why I set different audience offset.